[2022-01-07 15:35:53,271] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: new_description.print_date scheduled__2022-01-06T00:00:00+00:00 [queued]>
[2022-01-07 15:35:53,290] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: new_description.print_date scheduled__2022-01-06T00:00:00+00:00 [queued]>
[2022-01-07 15:35:53,290] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-01-07 15:35:53,290] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-01-07 15:35:53,291] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-01-07 15:35:53,332] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): print_date> on 2022-01-06 00:00:00+00:00
[2022-01-07 15:35:53,337] {standard_task_runner.py:52} INFO - Started process 89879 to run task
[2022-01-07 15:35:53,348] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'new_description', 'print_date', 'scheduled__2022-01-06T00:00:00+00:00', '--job-id', '323', '--raw', '--subdir', 'DAGS_FOLDER/new_description_dag.py', '--cfg-path', '/var/folders/3z/fyvld5jd6lxbb7nt49h4jx1h0000gn/T/tmp7383dugy', '--error-file', '/var/folders/3z/fyvld5jd6lxbb7nt49h4jx1h0000gn/T/tmpzsa8k_pt']
[2022-01-07 15:35:53,351] {standard_task_runner.py:77} INFO - Job 323: Subtask print_date
[2022-01-07 15:35:53,419] {logging_mixin.py:109} INFO - Running <TaskInstance: new_description.print_date scheduled__2022-01-06T00:00:00+00:00 [running]> on host 1.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.ip6.arpa
[2022-01-07 15:35:53,475] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=airflow@example.com
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=new_description
AIRFLOW_CTX_TASK_ID=print_date
AIRFLOW_CTX_EXECUTION_DATE=2022-01-06T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-06T00:00:00+00:00
[2022-01-07 15:35:53,477] {subprocess.py:62} INFO - Tmp dir root location: 
 /var/folders/3z/fyvld5jd6lxbb7nt49h4jx1h0000gn/T
[2022-01-07 15:35:53,478] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'date']
[2022-01-07 15:35:53,487] {subprocess.py:85} INFO - Output:
[2022-01-07 15:35:53,501] {subprocess.py:89} INFO - Fri Jan  7 15:35:53 IST 2022
[2022-01-07 15:35:53,501] {subprocess.py:93} INFO - Command exited with return code 0
[2022-01-07 15:35:53,533] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=new_description, task_id=print_date, execution_date=20220106T000000, start_date=20220107T100553, end_date=20220107T100553
[2022-01-07 15:35:53,577] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-01-07 15:35:53,603] {local_task_job.py:264} INFO - 2 downstream tasks scheduled from follow-on schedule check
